src/Parser/Monk/Brewmaster/Modules/Features/Abilities.js

```diff
@@ -43,14 +43,6 @@ class Abilities extends CoreAbilities {
         spell: SPELLS.TIGER_PALM,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
       },
-      {
-        spell: SPELLS.EXPLODING_KEG,
```

Comment:
You could have the `enabled:` check if the artifact is equipped instead of removing it.